### PR TITLE
Hasan/Translation 1044 dsi indices

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/instruments/_drift_switch_indices.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Symbol from '../components/helper/_symbol'
+import { drift_switch_indices } from '../static/content/_market-symbols'
+
+const DriftSwitchIndices = () => {
+    return (
+        <>
+            {drift_switch_indices.map((symbol, index) => (
+                <Symbol key={index} src={symbol.src} text={symbol.text} />
```
Comment (habib-deriv) [inline src/pages/markets/instruments/_drift_switch_indices.tsx:9]: Can remove the `index` and use` symbol.text` as key

File: src/pages/markets/static/content/_cfds.tsx [hunk 2]
```diff
@@ -119,6 +121,12 @@ export const synthetic_cfds: TMarketContent = {
         mobile_col: 2,
     },
     content: [
+        {
+            id: 'drift-switch-indices',
+            title: '_t_Drift switch indices_t_',
```
Comment (konstantinosG-deriv) [inline src/pages/markets/static/content/_cfds.tsx:126]: Minor issue: In figma these titles have Ubuntu font family, so might have to change these and the rest

**Review by mayuran-deriv — APPROVED** [review]

File: src/pages/markets/static/content/_cfds.tsx (hunk 2 above)
Comment (hasan-deriv) [reply src/pages/markets/static/content/_cfds.tsx:126]: @konstantinosG-deriv, should i change the font family?

File: src/pages/markets/instruments/_drift_switch_indices.tsx (hunk 1 above)
Comment (hasan-deriv) [reply src/pages/markets/instruments/_drift_switch_indices.tsx:9]: Sure

File: src/pages/markets/static/content/_cfds.tsx (hunk 2 above)
Comment (hasan-deriv) [reply src/pages/markets/static/content/_cfds.tsx:126]: done

**Review by konstantinosG-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Pull request merged**